android/build.gradle

```diff
@@ -1,21 +1,41 @@
+// Top-level build file where you can add configuration options common to all sub-projects/modules.
+
 buildscript {
     ext {
-        buildToolsVersion = "34.0.0"
-        minSdkVersion = 33
-        compileSdkVersion = 34
-        targetSdkVersion = 34
+        buildToolsVersion = findProperty('android.buildToolsVersion') ?: '34.0.0'
+        minSdkVersion = Integer.parseInt(findProperty('android.minSdkVersion') ?: '23')
+        compileSdkVersion = Integer.parseInt(findProperty('android.compileSdkVersion') ?: '34')
+        targetSdkVersion = Integer.parseInt(findProperty('android.targetSdkVersion') ?: '34')
+        kotlinVersion = findProperty('android.kotlinVersion') ?: '1.9.23'
```

Comment:
@NeOMakinG revert with `git checkout 29031539c159d035d5754de020c68035394bbd38 -- android/build.gradle` pls

Reply:
I didn't checkout the whole file but setup these numbers strictly to be as close as a raw project as possible